Sources/StreamChat/Utils/ChatHashable.swift

```diff
@@ -0,0 +1,137 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+protocol ChatHasher {
```

Comment:
I know this is not public API but some kind of basic documentation for our future selves would be nice.

Reply:
fixed in d1a4ed6f97a6a61cec7487f6fe07b07120e78617